scripts/check_urls.py

```diff
@@ -0,0 +1,205 @@
+"""
+Copyright 2022 Objectiv B.V.
+"""
+import datetime
+import sys
+import argparse
+import requests
+import xml.etree.ElementTree as ElementTree
+from typing import List, Dict
+import glob
+import urllib.parse
+from blessings import Terminal
+from subprocess import check_output
+
+# Basic configuration
+FQDN = 'https://objectiv.io' # domain to get sitemaps from

```

Comment:
```suggestion
FQDN = 'https://objectiv.io'  # domain to get sitemaps from

```
pep8 wants 2 spaces before the comment